### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PoojaChandak**
Observed a few changes. Suggesting the same. Kindly check.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,11 @@ Looking to see how you can create your own OpenAPI definition, present it, or ot
...
 
-The TSC holds weekly web conferences to review open pull requests and discuss open issues related to the evolving OpenAPI Specification. Participation in weekly calls and scheduled working sessions is open to the community. You can view the [TSC calendar online](https://openapi.groups.io/g/tsc/calendar), and import it to your calendar using the [iCal link](https://openapi.groups.io/g/tsc/ics/1105671/1995679554/feed.ics).
+The TSC holds weekly web conferences to review open pull requests and discuss open issues related to the evolving OpenAPI Specification. Participation in weekly calls and scheduled working sessions are open to the community. You can view the [TSC calendar online](https://openapi.groups.io/g/tsc/calendar), and import it to your calendar using the [iCal link](https://openapi.groups.io/g/tsc/ics/1105671/1995679554/feed.ics).
```
Comment (ioggstream) [inline README.md:47]: Participation ... *is* open

**Review by ioggstream — CHANGES_REQUESTED** [review]
see comments

File: README.md (hunk 1 above)
Comment (PoojaChandak) [reply README.md]: Made the change as per review comment.

**Comment by PoojaChandak** [comment]
The changes are done. Kindly check. @ioggstream

**Comment by PoojaChandak** [comment]
Gentle reminder. Kindly check.

**Review by chrissy35 — APPROVED** [review]

**Comment by darrelmiller** [comment]
Thank you for your suggestions.  I believe some of these grammar suggestions are subjective and don't bring a significant amount of additional clarity.  If you could provide some further evidence of why the current words should be changed we would consider making the changes.  We will take any guidance you can provide and use it for future work in this area.

**Pull request closed without merging**